### PR TITLE
add api key usage

### PR DESCRIPTION
**Pull request by stubendek**

### CODE REVIEW

File: serverless-editor-demo.html [hunk 1]
```diff
@@ -11,8 +11,13 @@
 	<v-app>
 	  <v-main>
 		<v-container>
+		  <v-text-field
+            v-model="apikey"
```
Comment (gyulanemeth) [inline serverless-editor-demo.html]: Tab issue

File: serverless-editor-demo.html [hunk 2]
```diff
@@ -35,33 +40,45 @@
 	  	vuetify: new Vuetify(),
 	  	data() {
 			return {
-				chamaileonPlugins: null
+				chamaileonPlugins: null,
+				apikey: null,
```
Comment (gyulanemeth) [inline serverless-editor-demo.html]: due to our naming conventions it should be "apiKey"

File: serverless-editor-demo.html [hunk 3]
```diff
@@ -35,33 +40,45 @@
...
-						'yellow': '#ffd23c',
-					},
-				},
-			})
-		},
 		methods: {
 			async openEditor() {
+
+				const accessTokenRequest = await fetch("https://sdk-api.chamaileon.io/api/v1/tokens/generate", {
+					method: "GET", // or 'PUT'
```
Comment (gyulanemeth) [inline serverless-editor-demo.html]: Please remove the // or 'PUT' comment

**Review by gyulanemeth — CHANGES_REQUESTED** [review]

File: serverless-editor-demo.html [hunk 4]
```diff
@@ -1,417 +1,429 @@
...
-						console.log('Email title has been updated to: ', title)
-						setTimeout(resolve, 1000) // simulating async
-					})
+<body>
+	<div id='app'>
+		<v-app>
+			<v-main>
+				<v-container>
+					<v-text-field
+						v-model="apiKey"
```
Comment (kadaradam) [inline serverless-editor-demo.html:17]: Please do not mix double and single quotes. We use single in the example script.

File: serverless-editor-demo.html [hunk 5]
```diff
@@ -1,417 +1,429 @@
...
+					async function onSave({ emailJson }) {
+						return new Promise(async function (resolve) {
+							console.log('Email saved. emailJson:', emailJson)
+							return resolve()
+						})
+					}
 
-						return resolve({
-							src: selectedItem.value
+					async function onAutoSave({	emailJson }) {
```
Comment (kadaradam) [inline serverless-editor-demo.html:128]: Tab

File: serverless-editor-demo.html [hunk 6]
```diff
@@ -1,417 +1,429 @@
...
+<body>
+	<div id='app'>
+		<v-app>
+			<v-main>
+				<v-container>
+					<v-text-field
+						v-model="apiKey"
+						counter="25"
+						label="API key"
+						unegat>
```
Comment (kadaradam) [inline serverless-editor-demo.html:20]: Is this a valid prop?

File: serverless-editor-demo.html [hunk 7]
```diff
@@ -1,417 +1,429 @@
...
+			methods: {
+				async openEditor() {
+
+					const accessTokenRequest = await fetch(
+					"https://sdk-api.chamaileon.io/api/v1/tokens/generate", {
+						method: "GET",
+						headers: {
+							"Authorization": `Bearer ${this.apiKey}`,
+						},
+					});
```
Comment (kadaradam) [inline serverless-editor-demo.html:56]: To keep the convention please remove `;`

**Review by kadaradam — APPROVED** [review]

**Pull request closed without merging**